### DISCUSSION TITLE
A demo site that use Jupyterlite

### DISCUSSION THREAD

**Discussion by ruoyu0088** [jupyterlite/jupyterlite#851] [Show and tell]
I recently found Jupyterlite, and ported many examples that can be executed in the Jupyterlite environment. Jupyterlite is so powerful, that didn't use JupyterLab as much as before.

link to the github repository: https://github.com/ruoyu0088/jupyterlite_demos
link to the demo site: https://ruoyu0088.github.io/jupyterlite_demos/lab/index.html

Here are some images:

![](https://github.com/ruoyu0088/jupyterlite_demos/raw/main/images/julia.gif)

![](https://github.com/ruoyu0088/jupyterlite_demos/raw/main/images/ants.gif)

![](https://github.com/ruoyu0088/jupyterlite_demos/raw/main/images/strandbeest_walk.gif)

![](https://github.com/ruoyu0088/jupyterlite_demos/raw/main/images/magnetcube.png)

**Comment by martinRenou** [comment]
Thanks for sharing! Those demos are super cool 😄
